organizations/organizations.go

```diff
@@ -109,8 +109,8 @@ func Create(w http.ResponseWriter, r *http.Request) {
 	// var creatorid interface{} = creator.ID
 	// var creatorid primitive.ObjectID = creator.ID
 	// var ccreatorid string = creatorid.(primitive.ObjectID).Hex()
-	var ccreatorid string = creator.ID
```

Comment:
Do this fixes made in this file impact your work? if no, please avoid doing such.